Source: Gaute Hope (c) <eg@gaute.vetsj.com>
Upstream: No
Reason: Stripping done by package manager
--- old/Makefile	2009-07-07 17:29:12.000000000 +0000
+++ work/Makefile	2009-10-14 08:40:22.000000000 +0000
@@ -40,8 +40,6 @@
 	@rm -rf dvtm-${VERSION}
 
 install: dvtm
-	@echo stripping executable
-	@strip -s dvtm
 	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
 	@mkdir -p ${DESTDIR}${PREFIX}/bin
 	@cp -f dvtm ${DESTDIR}${PREFIX}/bin
